src/gtimelog/data/org.gtimelog.gschema.xml

```diff
@@ -157,6 +157,12 @@
             <description>URL for editing tasks for the task pane.  Will be opened in a browser window if the user asks to edit tasks.</description>
         </key>
 
+        <key name="remote-auth-header" type="s">
+            <default>""</default>
+            <summary>Remote authentication token</summary>
+            <description>Authentication token for when task-list-url is protected</description>
```

Comment:
I think private tokens should rather be stored in the system keyring via SecretService rather than in gsettings.